src/prefect/cli/_prompts.py

```diff
@@ -394,13 +391,12 @@ def prompt_schedules(console) -> List[DeploymentScheduleCreate]:
                 "active": is_schedule_active,
             }
 
-            if PREFECT_EXPERIMENTAL_ENABLE_SCHEDULE_CONCURRENCY:
-                max_active_runs = prompt_for_schedule_max_active_runs(console)
-                catchup = prompt_for_schedule_catchup(console)
+            max_active_runs = prompt_for_schedule_max_active_runs(console)
+            catchup = prompt_for_schedule_catchup(console)
 
-                minimal_schedule_kwargs.update(
-                    {"max_active_runs": max_active_runs, "catchup": catchup}
-                )
+            minimal_schedule_kwargs.update(
+                {"max_active_runs": max_active_runs, "catchup": catchup}
+            )
```

Comment:
Hmm, should leave the flag in 3.x, @zzstoatzz ?

Reply:
I'm inclined to just leave the flag/code in place. If we're going to revisit it we'll need an experimental flag, and it'd be really sad if that was different than the 2.x line. @zzstoatzz if you feel strongly otherwise let me know.

Reply:
that sounds good to me!